lib/utils/requireModule/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+	"parser": "babel-eslint"
```

Comment:
This `parser` option is for only one purpose: correct parse the `import` statements.